lib/ansible/modules/cloud/vmware/vmware_vswitch.py

```diff
@@ -138,7 +150,11 @@ def state_create_vswitch(self):
         vss_spec.numPorts = self.number_of_ports
         vss_spec.mtu = self.mtu
         if self.nic_name:
-            vss_spec.bridge = vim.host.VirtualSwitch.BondBridge(nicDevice=[self.nic_name])
+            if isinstance(self.nic_name, list):
```

Comment:
this is not needed as you made nic_name a list in argspec, it will always be a list